CommentingPractice/model/snack.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file snack.js
+ * @author: Cameron Senor (Firebase)
+ */
+export default class Snack{
+
+    /**
+    * Constructor
+    * @para {String} typeOfSnack - The type of snack (fruit, candy, ect)
+    * @para {boolean} isHealthy - True if healthy, false if not.
+    */
+    constructor(typeOfSnack, isHealthy){
+    var type;
+    var heathly;
+    }//closes constructor
```

Comment:
You have a constructor taking two arguments here, but it does not use what is passed in to set the type or healthy property. You also need to attach those properties to the this object instead of declaring them as local variables within the scope of the constructor.